commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -190,6 +230,36 @@ public static CommerceOrder addCheckoutDetailsToUserOrder(
 		return CommerceOrderLocalServiceUtil.updateCommerceOrder(commerceOrder);
 	}
 
+	public static CommerceCatalog addCommerceCatalog(
+			long groupId, String commerceCurrencyCode)
+		throws Exception {
+
+		return CommerceCatalogServiceUtil.addCommerceCatalog(
+			RandomTestUtil.randomString(), commerceCurrencyCode,
+			LocaleUtil.US.getDisplayLanguage(), null,
```

Comment:
`LocaleUtil.US.getDisplayLanguage()` returns probably `English`, but this method expects a languageId. I think we need to change this (and any other instances) to `LocaleUtil.toLanguageId(LocaleUtil.US)`.